doc/source/user_guide/window.rst

```diff
@@ -198,6 +198,36 @@ from present information back to past information. This allows the rolling windo
 
    df
 
+.. _window.reverse_rolling_window:
+
+Reverse rolling window
```

Comment:
Can you just add this as an example of using the `FixedForwardWindowIndexer` in the `Custom window rolling` section below?

Reply:
Sorry for late reply. Sure i will add it.